.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+## Description
+
+Please briefly explain the changes you made here.
+
+## Jira Ticket
+
+Please add link to the Jira ticket.
+
+## Coding Standards
+
+https://gpitbjss.atlassian.net/wiki/spaces/NIA/pages/2108522539/Java+Coding+Standards
+
+## Checklist
+
+- [ ] Acceptance Criteria met
+- [ ] Branch named {feature|task|bug|hotfix}/{NIAD-*}
+- [ ] Commit messages are meaningful
+- [ ] Manually tested
+- [ ] All unit tests passing
+- [ ] All integration tests passing
+- [ ] Pulled latest main into feature branch
+- [ ] Code reviewed
+- [ ] If your pull request depends on any other, please link them in the description
+- [ ] main branch is passing/green on CI
```

Comment:
Some of these we can see from the github checks (Branch name, unit tests passing, integration tests passing, code reviewed) and others can become outdated (pulled latest main, main branch is green). One suggestion for re-wording. Would also suggest a tick for reviewing our own code prior to asking others to do so, and a reminder for any documentation tasks:

```suggestion
- [ ] Acceptance Criteria met
- [ ] Commit messages are meaningful
- [ ] Manually tested
- [ ] All PR dependencies linked in the description
- [ ] Reviewed own code
- [ ] Any required documentation updates are complete
```

Reply:
I think keeping documentation up to date is important, so keen for that checkbox to be added

Reply:
True. The GitHub checks obviously will never be removed, I included them in the checklist as it's still part of our "Ready to merge". But happy to take them out